pkg/apis/crds/karpenter.k8s.aws_awsnodetemplates.yaml

```diff
@@ -193,6 +193,61 @@ spec:
                       credentials are not available."
                     type: string
                 type: object
+              networkInterfaces:
+                items:
+                  properties:
+                    associateCarrierIPAddress:
+                      description: "Associates a Carrier IP address with eth0 for
+                        a new network interface. \n Use this option when you launch
+                        an instance in a Wavelength Zone and want to associate a Carrier
+                        IP address with the network interface. For more information
+                        about Carrier IP addresses, see Carrier IP addresses (https://docs.aws.amazon.com/wavelength/latest/developerguide/how-wavelengths-work.html#provider-owned-ip)
+                        in the Wavelength Developer Guide."
+                      type: boolean
+                    associatePublicIPAddress:
+                      description: Associates a public IPv4 address with eth0 for
+                        a new network interface.
+                      type: boolean
+                    deleteOnTermination:
+                      description: Indicates whether the network interface is deleted
+                        when the instance is terminated.
+                      type: boolean
+                    description:
+                      description: A description for the network interface.
+                      type: string
+                    deviceIndex:
+                      description: The device index for the network interface attachment.
+                      format: int64
+                      type: integer
+                    interfaceType:
+                      description: "The type of network interface. To create an Elastic
+                        Fabric Adapter (EFA), specify efa. For more information, see
+                        Elastic Fabric Adapter (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/efa.html)
+                        in the Amazon Elastic Compute Cloud User Guide. \n If you
+                        are not creating an EFA, specify interface or omit this parameter.
+                        \n Valid values: interface | efa"
+                      type: string
+                    ipv4PrefixCount:
+                      description: The number of IPv4 prefixes to be automatically
+                        assigned to the network interface. You cannot use this option
+                        if you use the Ipv4Prefix option.
+                      format: int64
+                      type: integer
+                    ipv6PrefixCount:
+                      description: The number of IPv6 prefixes to be automatically
+                        assigned to the network interface. You cannot use this option
```

Comment:
can you remove the last sentence since we're not supporting the `ipv6Prefix` property?